app/src/pages/participant/uploads.js

```diff
@@ -116,12 +117,23 @@ module.exports = function(params) {
             .obs('href','')
             .obs('collapsed', true)
             .obs('completedBy', '');
+            
+        item.isUploadRequested = "completed step";
         if (item.status === 'succeeded') {
             var id = item.schemaId;
             var rev = item.schemaRevision;
             item.contentObs(id);
             item.hrefObs('/#/schemas/'+encodeURIComponent(id)+'/versions/'+rev);
+            item.isUploadCompleted = "completed step";
+        } else {
+            item.isUploadCompleted = "active step";
         }
+        if (item.status === 'succeeded' && item.healthRecordExporterStatus === 'succeeded') {
+            item.isExportCompleted = "completed step";  
+        } else {
+            item.isExportCompleted = "active step";
+        }
+        
         item.completedByObs(formatCompletedBy(item));
     }
     function formatCompletedBy(item) {
```

Comment:
![bridge_study_manager-2](https://cloud.githubusercontent.com/assets/5589085/19404931/0c54196c-9227-11e6-9e9b-b3f6edace76e.png)

Has checkmarks rather than showing progression over time, doesn't have the little arrow indentations to show steps, is way too large, and has a 3 in the final slot.